playbooks/09 - ChatOps/Bot Command - Get Reputation.json

```diff
@@ -344,16 +344,11 @@
     ],
     "priority": null,
     "uuid": "cc0e8cf6-b481-4138-b548-e0b594b407c7",
+    "owners": [],
+    "isPrivate": false,
+    "deletedAt": null,
     "recordTags": [
         "Subroutine",
         "Bot - Execute commands"
-    ],
-    "id": "741",
-    "createUser": "\/api\/3\/people\/3451141c-bac6-467c-8d72-85e0fab569ce",
```

Comment:
why we removed this?

Reply:
@cs-suyog-jinturkar These keys got removed in build 900 from playbooks json, so these keys got removed automatically when performed export

Reply:
Can we please track this change for all other SPs as well?

Reply:
@cs-suyog-jinturkar this particular change is only reflected when we make any change in the playbook. otherwise, playbook JSON remains the same.